core/src/main/java/com/taobao/arthas/core/command/monitor200/MonitorCommand.java

```diff
@@ -45,6 +47,12 @@ public void setMethodPattern(String methodPattern) {
         this.methodPattern = methodPattern;
     }
 
+    @Argument(argName = "condition-express", index = 2, required = false)
+    @Description("Condition-express to filter method who need statistics")
```

Comment:
Condition express to filter method calls

Reply:
changed the desc